### DISCUSSION TITLE
create-remix support for HTTP proxies

### DISCUSSION THREAD

**Discussion by guigui64** [remix-run/remix#2526] [Proposals]
Scripts such as `create-remix` use `node-fetch` for downloading the templates from the Github API.
Unfortunately, `node-fetch` does not support HTTP(S) proxies.

However, there are some elegant solutions to this such as using `global-agent`:
https://github.com/gajus/global-agent

That way, the users of the script will only need to export environment variables to be able to use them behind proxies.

**Comment by cliffordfajardo** [comment]
@guigui64 - Does this recent pull request solve your proposed solution?
[feat(create-remix): Add proxy support in create cli #4159](https://github.com/remix-run/remix/pull/4159)
